src/pages/ContentScripts/features/developer-avator-openrank/index.tsx

```diff
@@ -0,0 +1,66 @@
+import features from '../../../../feature-manager';
+import { getOpenrank } from '../../../../api/developer';
+import elementReady from 'element-ready';
+
+const featureId = features.getFeatureID(import.meta.url);
+
+const getData = async (developerName: string): Promise<string | null> => {
+  const data = await getOpenrank(developerName);
+  if (data) {
+    const keys = Object.keys(data);
+    const latestKey = keys[keys.length - 1];
+    return data[latestKey];
+  }
+  return null;
+};
+
+const getDeveloperName = (target: HTMLElement): string | null => {
+  const hovercardUrl = target.getAttribute('data-hovercard-url');
+  if (!hovercardUrl) return null;
+
+  const matches = hovercardUrl.match(/\/users\/([^\/]+)\/hovercard/);
+  return matches ? matches[1] : null;
+};
+
+const init = async (): Promise<void> => {
+  // 监听具有 data-hovercard-type="user" 属性的元素
+  document.querySelectorAll('[data-hovercard-type="user"]').forEach((element) => {
+    element.addEventListener('mouseover', async () => {
+      // 获取开发者名称
+      const developerName = getDeveloperName(element as HTMLElement);
+      if (!developerName) {
+        console.error('Developer name not found');
```

Comment:
Maybe we should remove the `console.error` here and other two places (L 39 and L57). It is triggered so often and making the console message a little bit scary.

Reply:
already edited